admin/views/access-arrangements/overview.ejs

```diff
@@ -55,7 +55,45 @@
         </form>
     </div>
     <div class="column-two-thirds">
+        <%if (pupils.length > 0) { %>
+        <table id='submitted-pupil-access-arrangements'>
+            <thead>
+            <tr>
+                <th scope="col">Pupil</th>
+                <th scope="col">Arrangement</th>
+            </tr>
+            </thead>
+            <tbody>
+            <% pupils.forEach( (pupil) => { %>
+            <tr>
+                <td class="access-arrangements-pupil-name">
+                    <div<% if (highlight && highlight.includes(pupil.urlSlug)) { %> class="highlight-item"<% } %>>
+                        <div id="<%= pupil.urlSlug %>">
+                        <%= pupil.fullName %>
+                            <% if (pupil.showDoB) { %>
+                            <div class="font-greyed-out font-xsmall">
+                                Date of birth: <%= pupil.dateOfBirth %>
+                            </div>
+                            <% } %>
+                        </div>
+                    </div>
+                </td>
+                <td class="pupil-access-arrangements">
+                    <% pupil.arrangements && pupil.arrangements.forEach((pupilAccessArrangement, i) => { %>
+                        <%if (pupilAccessArrangement.length - 1 !== i && pupilAccessArrangement.length > 1) { %>
+                            <span><%= pupilAccessArrangement %></span><br />
+                        <% } else { %>
+                            <span><%= pupilAccessArrangement %></span>
+                        <% } %>
```

Comment:
I think this will be better if it’s just a `<div><%= pupilAccessArrangement %></div>` instead of the conditional, since the div has `display: block` (so, line breaks) by default — also, I think you meant to use `pupil.arrangements` instead of `pupilAccessArrangement` in the if statement? (since now it compares the access arrangement length instead of the list length)

Other than that, I think it looks okay 👍